app/src/main/res/values/strings.xml

```diff
@@ -548,6 +556,11 @@
     <string name="reading_list_articles_added_to_named">Added %1$d articles to %2$s</string>
     <string name="reading_list_articles_moved_to_named">Moved %1$d articles to %2$s</string>
     <string name="reading_list_delete_confirm">Are you sure you want to delete %s?</string>
+    <string name="reading_list_delete_lists_confirm_dialog_title">Delete selected list(s)?</string>
```

Comment:
Would it be better if we have this title supports plurals?

Reply:
This was initially a plural.. then we changed to this ..I think Robin is trying to give strings like these to avoid plurals in some places where plurals aren't too important..